services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java

```diff
@@ -65,6 +65,16 @@
    public static final String TEA_HOUSES_TBL_NAME = "tea_houses";
    public static final String COLUMN_ID1_VALUE_ONE = "ayy lmao";
    public static final int COLUMN_ID2_VALUE_ONE = 3;
+   public static final double TEST_DECIMAL_ONE = 3.1415;
+   public static final double TEST_DECIMAL_TWO = 9.5;
+   public static final double TEST_DECIMAL_THREE = 9.813793;
+   public static final int TEST_INT_ONE = 16;
+   public static final int TEST_INT_TWO = 15;
+   public static final int TEST_INT_THREE = 18;
+   public static final int TEST_INT_FOUR = 2;
+   public static final String TEST_STRING_ONE = "test2";
+   public static final String TEST_STRING_TWO = "columnId = ?";
```

Comment:
This is actually the where clause so the name COLUMN_ID_WHERE_CLAUSE is more appropriate, it is not a test string

Reply:
here is some info https://www.w3schools.com/sql/sql_where.asp

Reply:
Thank you. I understand now. I have renamed it.